js/components/SetAReminder/SetAReminder.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import {
+  Text,
+  View,
+  Modal,
+  TouchableOpacity,
+  Switch,
+  StyleSheet,
+  DatePickerIOS
+} from "react-native";
+import LinearGradient from "react-native-linear-gradient";
+import styles from "./styles";
+
+class SetAReminder extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      chosenDate: new Date(),
+      chosenTime: "8:30",
+      modalVisible: false,
+      reminderSwitch: false
+    };
+  }
+  setDate = newDate => {
+    console.log(newDate)
+    this.setState({ chosenDate: newDate });
+  };
+  toggleReminderSwitch = value => {
+    this.setState({ reminderSwitch: value });
+  };
+  setModalVisible(visible) {
```

Comment:
Stick to one style of class method. If you're using static methods with arrows, make sure all your methods are as such.